aries_vcx/src/messages/basic_message/message.rs

```diff
@@ -15,9 +17,13 @@ pub struct BasicMessage {
     pub l10n: Option<Localization>,
     #[serde(rename = "~thread")]
     pub thread: Option<Thread>,
+    #[serde(rename = "~timing")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub timing: Option<Timing>,
```

Comment:
Yes, basic message 1.0 protocol was created at the time when `~timing` decorator (probably the concept of aries decorators) did not exist yet. I'd suggest to keep it nevertheless, `~timing` decorator is much more powerful (it can contain whole bunch of additional timing information)